doc/configuration.rst

```diff
@@ -91,25 +88,25 @@ Security
 --------
 
 It is a good practice to have the cookies require a https connection for
-security reason. However, while developing this can prevent the authentication
+reasons of security. However, while developing, this can prevent the authentication
 from working. So by default this is turned off to provide an out-of-the-box
 working configuration, however you will want to change it in production.
 
-The setting to change is ``SESSION_COOKIE_SECURE``.
+The setting is ``SESSION_COOKIE_SECURE``.
 
 **Default** ``SESSION_COOKIE_SECURE =  False``
 
-To change to ``SESSION_COOKIE_SECURE = True``.
+Change this ``SESSION_COOKIE_SECURE = True`` when using the application in a production environment.
```

Comment:
I was kind of looking at it as almost one sentence: The setting is XX, default XX=YY, to change to XX=AA but looking at it again, it makes sense to split it over two sentences (and I like the precision added at the end).